e2e/features/submit-hearing-requirements.feature

```diff
@@ -48,7 +47,6 @@ Feature: Submit hearing requirements
 
     When I click the `Appellant` tab
     Then I should see the appellant's details
-    And I should see the legal representative's details
```

Comment:
Same as above?

Reply:
Added - should see the legal representative details